lib/ansible/module_utils/cloudscale.py

```diff
@@ -88,3 +93,13 @@ def _delete(self, api_call):
         else:
             self._module.fail_json(msg='Failure while calling the cloudscale.ch API with DELETE for '
                                        '"%s".' % api_call, fetch_url_info=info)
+
+    def get_returns(self, resource):
+        if resource:
+            for k, v in resource.items():
+                if k in self._transforms:
+                    new_key = self._transforms[k]['to_key']
+                    self._result[new_key] = v
```

Comment:
As already noted elsewhere I would prefer the transformations to be done as early as possible and not as late as possible just before returning the data because it makes the code easier to understand.